### DISCUSSION TITLE
docs/a-feast-for-crows/

### DISCUSSION THREAD

**Discussion by giscus** [Yarden-zamir/me#16] [Announcements]
# docs/a-feast-for-crows/

https://www.yarden-zamir.com/docs/a-feast-for-crows/

**Comment by Yarden-zamir** [comment]
That's the problem with audiobooks, I simply don't take notes, it's not a part of the gameplay loop

**Reply by Yarden-zamir** [reply]
Maybe I should just not have those in the first place, scoring a piece of media may not be enough to justify a note.

That beeing said, having more notes was never a problem before. They are only a problem now because people navigating through are stumbling into them.
